src/components/Button/PrimaryButton.tsx

```diff
@@ -5,14 +5,19 @@ const PrimaryButton = styled.div<{
   background?: string;
   marginTop?: number;
   marginBottom?: number;
+  color?: string;
+  display?: string;
+  justifyContent?: string;
 }>`
   background: ${({ background, theme }) => background || theme.color.background.button};
   padding: 23px;
   border-radius: 16px;
   ${({ marginTop }) => !!marginTop && `margin-top: ${marginTop}px;`}
   ${({ marginBottom }) => !!marginBottom && `margin-bottom: ${marginBottom}px;`}
+  ${({ display }) => !!display && `display: ${display};`}
+  ${({ justifyContent }) => !!justifyContent && `justify-content: ${justifyContent};`}
   font-size: 20px;
-  color: ${({ theme }) => theme.color.text.button};
+  color: ${({ color, theme }) => color || theme.color.text.button};
```

Comment:
I think `color **??** theme.color.text.button` would work here as if I am not mistaken the only check needed here is if color is undefined or not right? We may not check falsy.